scripts/drop_issuer_nf_tokens.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash 
```

Comment:
i _personally_ would prefer a convention of using python, just because bash is hard for me to write. but i'll let CJ weigh in